config/init.go

```diff
@@ -110,6 +110,10 @@ const DefaultConnMgrGracePeriod = time.Second * 20
 // type.
 const DefaultConnMgrType = "basic"
 
+// DefaultResourceMgrMinInboundConns is a MAGIC number that probably a good
```

Comment:
This value should be where it is used, so I would move it to rcmgr.go

Reply:
This was proposed by @lidel https://github.com/ipfs/kubo/pull/9555#discussion_r1071441852
As well as we expect this value to be used in #9548

Reply:
We have other defaults in `config` package, this makes it consistent.